src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -144,6 +144,31 @@
 short-summary: Manage vault network ACLs.
 """
 
+helps['keyvault private-endpoint'] = """
+type: group
+short-summary: Manage vault private endpoint connections.
+"""
```

Comment:
On second thought, we'd better name it as `private-endpoint-connection` as it corresponds to  `privateEndpointConnections` from `az keyvault show`.

```jsonc
    "privateEndpointConnections": [
      {
        "etag": "9fae19d5c0704012a6b0544574fe24d2",
        "id": "/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/rg1/providers/Microsoft.KeyVault/vaults/kv0130/privateEndpointConnections/c9e56b4f661f4cf382aeb5d832bf6f9f",
        "privateEndpoint": {
          "id": "/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/rg1/providers/Microsoft.Network/privateEndpoints/pe1",
          "resourceGroup": "rg1"
        },
        "privateLinkServiceConnectionState": {
          "actionRequired": null,
          "actionsRequired": "None",
          "description": "",
          "status": "Approved"
        },
        "provisioningState": "Succeeded",
        "resourceGroup": "rg1"
      }
```

`privateEndpoint` stands for another resource type `Microsoft.Network/privateEndpoints`. I think we shouldn't mix these two concepts.

Reply:
@jiasli Thanks for this, for the convenience, I shortened the group name as `private-endpoint` (in fact, I copied the group name from `network private-endpoint`), but as you mentioned, this name stands for another resource type and may cause confusion, I will modify this later.